reverse_proxy/templates/generic.conf.j2

```diff
@@ -13,7 +13,7 @@ server {
   location /{{ generic_name }}-fetch/{{ eicar_path }} {
     internal;
     rewrite_by_lua_block {
-      ngx.say("X5O!P%@AP[4\\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*")
+      ngx.print("X5O!P%@AP[4\\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*")
```

Comment:
The real clamav with production DB only detects eicar without a trailing newline.